src/pages/transactions/index.vue

```diff
@@ -9,20 +9,20 @@
       {{ transactionsHints.transaction }}
     </template>
   </page-header>
-  <template v-if="!isLoading">
-    <transactions-statistics class="transactions-panel"/>
-    <transactions-chart-panel
-      :has-select="false"
-      :range="CHART_INTERVALS_OPTIONS[0]"
-      class="transactions-panel"/>
-    <transactions-panel class="transactions-panel"/>
-  </template>
+  <transactions-statistics class="transactions-panel"/>
```

Comment:
united with accounts now, i improved the content jumping when loading. Still, some more work to be done, but much better now.